test_wikidata_structure.py

```diff
@@ -38,6 +38,7 @@ def is_not_a_behavior(self, type_id):
         self.is_not_a_specific_error_class(type_id, 'a human behavior')
 
     def is_not_a_specific_error_class(self, type_id, expected_error_class):
+        wikimedia_connection.set_cache_location(osm_handling_config.get_wikimedia_connection_cache_location())
```

Comment:
is it needed?

Reply:
I had issues running a single test in isolation.

I have pushed another fix where it is just using the setup_module https://docs.pytest.org/en/7.1.x/how-to/xunit_setup.html

Reply:
oh, so you were using IDE that can trigger single test, right?

Reply:
Yes I am using PyCharm.

I have also executed all the tests as part of this change and no new failures